batch/batch/utils.py

```diff
@@ -69,15 +69,15 @@ def worker_memory_per_core_gb(worker_type):
 
 def worker_memory_per_core_bytes(worker_type):
     m = worker_memory_per_core_gb(worker_type)
-    return int(m * 1024**3)
+    return math.ceil(m * 1024**3)
```

Comment:
The smallest billable unit of memory we use in our system is MiB, so it doesn't make sense to talk about finer gradations of memory. For example, the quantity in the resources table is an integer, so we can't possibly bill fractions of an MiB.

We should define all memory in terms of this. We avoid ceilings and floating point math generally. We only need to convert to Bytes for Docker when we create a container specification.
```python3
def worker_memory_per_core_mib(worker_type):
    if worker_type == 'standard':
        return 3840
    elif worker_type == 'highmem':
        return 6656
    assert worker_type == 'highcpu', worker_type
    return 921
```

Reply:
Why did you decide on 921 instead of 922?